cassandra/src/test/scala/com/stratio/crossdata/connector/cassandra/CassandraImportTablesIT.scala

```diff
@@ -0,0 +1,167 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stratio.crossdata.connector.cassandra
+
+import com.datastax.driver.core.{Cluster, Session}
+import org.apache.spark.sql.crossdata.ExecutionType._
+import org.apache.spark.sql.crossdata.exceptions.CrossdataException
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class CassandraImportTablesIT extends CassandraWithSharedContext {
+
+  it should "import all tables from a keyspace" in {
+    assumeEnvironmentIsUpAndRunning
+
+    def tableCountInHighschool: Long = ctx.sql("SHOW TABLES").count
+    val initialLength = tableCountInHighschool
+
+    val importQuery =
+      s"""
+          |IMPORT TABLES
+          |USING $SourceProvider
+          |OPTIONS (
+          | cluster "$ClusterName",
+          | spark_cassandra_connection_host '$CassandraHost'
+          |)
+      """.stripMargin
+
+    val importedTables = ctx.sql(importQuery)
+
+    importedTables.schema.fieldNames shouldBe Array("tableIdentifier")
+    importedTables.collect.length should be > 0
+
+    // TODO We need to create an unregister the table
+    tableCountInHighschool should be > initialLength
+
+  }
+
+  it should "infer schema after import all tables from a Cassandra Database" in {
+    assumeEnvironmentIsUpAndRunning
+
+    val (cluster, session) = createOtherTables
+    val importQuery =
+      s"""
+         |IMPORT TABLES
+         |USING $SourceProvider
+         |OPTIONS (
+         | cluster "$ClusterName",
+         | spark_cassandra_connection_host '$CassandraHost'
+         |)
+      """.stripMargin
+
+    try {
+      sql(importQuery)
+
+      ctx.dropTempTable(s"$Catalog.$Table")
+
+      ctx.tableNames().size should be > 1
+      ctx.table(s"$Catalog.$Table").schema should have length 5
+    } finally {
+      cleanOtherTables(cluster, session)
+    }
+  }
+
+  it should "infer schema after import all tables from a keyspace" in {
+    assumeEnvironmentIsUpAndRunning
+
+    val (cluster, session) = createOtherTables
+
+    val importQuery =
+      s"""
+      |IMPORT TABLES
+      |USING $SourceProvider
+      |OPTIONS (
+      | cluster "$ClusterName",
+      | keyspace "$Catalog",
+      | spark_cassandra_connection_host "$CassandraHost"
+      |)
+    """.stripMargin
+
+    try {
+      ctx.dropAllTables()
+      sql(importQuery)
+      ctx.tableNames().size should be (2)
```

Comment:
Instead of checking the size( the ITs will be parallelized soon), the sql(importQuery) returns a Dataframe containing the imported tables.